src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { Card, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { BrowserRouter as Router, Route, Redirect, Link } from "react-router-dom";
+import axios from 'axios';
+
+export function ProfileView(props) {
+    console.log('props: ', props);
+
+    const [userData, setUserData] = useState({});
+
+    console.log('fav movies');
+
+    function getUserData(token) {
+        console.log('get user data');
+        axios.get('https://morning-badlands-52426.herokuapp.com/users/' + props.user, {
+            headers: { Authorization: `Bearer ${token}` }
+        })
+            .then(response => {
+                console.log('user', response.data);
+                // Assign the result to the state
+                setUserData(response.data);
+            })
+            .catch(function (error) {
+                console.log(error);
+            });
+    }
+
+    useEffect(() => {
+        console.log('use effect');
+        getUserData(localStorage.getItem('token'))
+    }, [])
+
+
+    return (
+        < Card >
+            <Card.Body>
+                <Card.Title>Profile Name: {userData.Username}</Card.Title>
+                <Card.Text>Email: {userData.Email}</Card.Text>
+                <Card.Text>Date 0f Birth: {userData.Birthday}</Card.Text>
+                <Card.Text>Favourite Movies: {userData.FavouriteMovies}</Card.Text>
```

Comment:
🔧`FavouriteMovies` is an array. To display its content, we shall have to loop through the array and display the movies one by one. React is not smart enough to pull the content out for you.

For now, I would comment it out until we have built out how to populate favorite movies and have them returned as part of the user profile info.

Reply:
Movies now displayed in user profile after being added to favourites.
Also removed from list after Remove button pressed.
However there seems to be an issue with the Heroku so I worked on heroku local.

Reply:
Think we went passed this issue